blocks/productcards/productcards.js

```diff
@@ -23,4 +23,6 @@ export default function decorate(block) {
     const text = document.createTextNode(link.textContent);
     body.replaceChild(text, link);
   });
+  const accessoriesTitle = document.querySelector('h1');
```

Comment:
This breaks when the document is not the accessories card. Check the console log of this page:

https://issue-56--airlessco--hlxsites.hlx.page/na/en/products/